ros2_batch_job/linux_batch/__init__.py

```diff
@@ -61,9 +61,13 @@ def setup_env(self):
         # but the installed official packages
```

Comment:
Is it _necessary_ now to source the bash script to setup connext properly if using debian installation? either way, this wording should probably be updated

Reply:
> Is it necessary now to source the bash script to setup connext properly if using debian installation?

Yes, still necessary. The files are being installed to `/opt/rti`.

> either way, this wording should probably be updated

Please propose a wording you have in mind and feel free to commit it directly to this PR.

Reply:
sounds like it needs to be removed completely then

Reply:
Updated in https://github.com/ros2/ci/pull/181/files#diff-eb845129d501bf69837f0bd51ec671eeR60